content/sensu-go/5.18/installation/install-sensu.md

```diff
@@ -144,44 +159,18 @@ For a complete list of configuration options, see the [backend reference][6].
 
 ### 3. Initialize
 
+{{% notice note %}}
+**NOTE**: If you are using Docker, you already completed intitialization in [2. Configure and start](#2-configure-and-start).
+Skip ahead to [4. Open the web UI](#4-open-the-web-ui) to continue the backend installation process.
```

Comment:
WDYT about using this step to admonish folks to change their password if they used Docker defaults?

Reply:
Yes, that's a good idea. I'll update to add it.